composer.json

```diff
@@ -70,18 +52,21 @@
     }
   },
   "require": {
-    "php": "^7.0",
+    "php": "^7.1",
     "symfony/config": "~2.8 || ~3.0",
     "symfony/dependency-injection": "~2.8 || ~3.0",
     "symfony/http-kernel": "~2.8 || ~3.0",
-    "prooph/event-store": "^6.3"
+    "prooph/event-store": "^7.0"
   },
   "require-dev": {
+    "prooph/event-sourcing": "dev-develop as 5.0.0",
+    "prooph/snapshot-store": "dev-master",
+    "prooph/pdo-event-store": "dev-master",
     "phpunit/phpunit": "^5.3",
     "phpunit/php-invoker": "^1.1.4",
```

Comment:
I needed the eventsourcing interfaces for the mock objects. I dont like it either. 

After this PR there will follow more. It is not meant as a new stable yet. 
See discussion on prooph private chat today with @rabbl